src/Collage/Render.elm

```diff
@@ -330,3 +342,56 @@ decodeDashing ds =
   ds
     |> List.map decodeOnOff
     |> String.join " "
+
+
+-- BASED ON https://github.com/rough-stuff/rough/blob/e9b0fdf36952a7a0f02e8015f4abac1ad39981c5/src/renderer.ts#L367
+bezierCurvePath : List Point -> String
+bezierCurvePath ps =
+    let
+        neighbors arr index =
+            Maybe.map4
+                (\m1 p1 i p2 ->
+                    ((m1, p1), (i, p2))
+                )
+                (Array.get (index - 1) arr)
+                (Array.get (index + 1) arr)
+                (Array.get index arr)
+                (Array.get (index + 2) arr)
+    in
+    case ps of
+        [] ->
+            ""
+
+        [ p ] ->
+            ""
+
+        [ (x1, y1), (x2, y2) ] ->
+            [ "M", [ x1, y1 ] |> List.map (String.fromFloat) |> String.join " "
+            , "L", [ x2, y2 ] |> List.map (String.fromFloat) |> String.join " "
+            ] |> String.join " "
+
+        (x1, y1) :: tail ->
+           let
+               last = List.reverse tail |> List.take 1
+               arr = Array.fromList ((x1, y1) :: (x1, y1) :: tail ++ last)
+               curves =
+                    Array.indexedMap (\i p ->
+                       case (neighbors arr i) of
```

Comment:
I struggle a bit to see why you need to double the first and last elements and use `neighbors` afterwards. If I understand it correctly, when looking at the original list of points:
* the first element needs `previous` to be the same point
* all other elements need the previous, sequent and subsequent points
* the final but last element needs needs its subsequent to be the same as the last
* the last element will be ignored
Is that correct?

You could then achieve the same result by mapping over the original list, converted to an array, and define neighbors as follows:
```elm
neigbors i xs = -- use array as last argument
  let
    l = length xs - 1
  in
  map4 (\m1 p0 p1 p2 -> (m1, p0, (p1, p2)))
  (get (max (i - 1) 0) xs) -- previous with first element as lower bound
  (get i xs)
  (get (max (i + 1) l) xs) -- sequent with last element as upper bound
  (get (i + 2) xs) -- if subsequent doesnt exist, we're ready and results in Nothing
```

Reply:
I think that's correct. The curve just needs to include the points at index `1` and index `length - 1`. I plugged in your solution but modified it slightly so it won't return `Nothing` for index `length - 1`.